### PR TITLE
Update to Calico 2.6.2

### PR DESCRIPTION
**Pull request by tmjd**
This PR also upgrades Flannel the same as #112 so I can rebase this when that PR is merged.

### CODE REVIEW

File: k8s-install/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 This directory includes manifests for deploying canal on Kubernetes using the Kubernetes API.  
 
-#### For Kubernetes 1.7
+#### For Kubernetes 1.7 and 1.8
```
Comment (caseydavenport) [inline k8s-install/README.md]: Perhaps "Kubernetes 1.7.+"?

v1.9. is just around the corner.

Comment (tmjd) [reply k8s-install/README.md]: I think I'd prefer being explicit and saying `For Kubernetes 1.7, 1.8, and 1.9`, that way users aren't wondering, when was the last time was this updated (what was out when this was updated)?

Comment (tmjd) [reply k8s-install/README.md]: WDYT?

Comment (caseydavenport) [reply k8s-install/README.md]: SGTM. Make sure you remember to update when v1.9 is released.

**Pull request merged**